### ISSUE TITLE
Images haven't access if UI disabled

### ISSUE THREAD

**Issue by dmitrydrynov** [keystonejs/keystone#6686]
Hi everyone,
When I disable Admin UI (ui.isDisabled = true) public images don't work
On url http://localhost:8000/images/37df2e62-bc65-425f-830c-60e4070d8233.png I see "Keystone is loading...". When Admin UI is enabled I see this image.
How I can work with public images, files when Admin UI is disabled?

**Comment by bjoluc** [comment]
Possibly related: When I try to `curl` the public image URL, I get
```
Found. Redirecting to /signin?from=...
```
According to the description "The base of the URL local images will be served from, outside of keystone." in the [image config docs](https://keystonejs.com/docs/apis/config#images), should we be serving images ourselves (e.g. via the `express.static` middleware)?

If I do so via
```ts
server: {
  extendExpressApp: (app, createContext) => {
    app.get("/test", express.static("public"));
  },
},
```
and I `curl` `/test/...`, Keystone still redirects to the signin page.

**Comment by mitchellhamilton** [comment]
This was fixed by #6893 which is available in `@keystone-next/keystone@27.0.2` and above.
